makefiles/boards/alias.inc.mk

```diff
@@ -0,0 +1,10 @@
+# list of known alias
+ALIAS.phynode   = pba-d-01-kw2x
+#TODO remove this line (as its just for show)
+ALIAS.linux     = native
+
+# this temporary variable is needed as we can not use BOARD when assigning BOARD
```

Comment:
I don't see how this can work. How about filling a list of existing alias on a per board basis and if the BOARD name is found in this list, use the alias trick ? (as I suggested in one of my previous comment)

Reply:
for one: it does :-)
second: this basically checks the list of defined aliases above, and if it finds a fitting entry the board variable is substituted. I think makes calls this concept `Computed Variable Names`

Reply:
I don't understand why you think this doesn't work?
All of the alias definitions are on the lines above.
If `BOARD` is set to `linux`, then `ALIAS_TARGET` will be assigned the value `$(ALIAS.linux)`, which evaluates to `native`.

Reply:
Oh ok, now I have it.